setup.py

```diff
@@ -100,7 +100,11 @@ def get_version():
     for line in version_file:
         if line.startswith("__version__"):
             version_tuple = ast.literal_eval(line.split(" = ")[1])
-            return ".".join(map(str, version_tuple))
+            version_str = ".".join(map(str, version_tuple))
```

Comment:
I picked your changes from https://github.com/elastic/apm-agent-python/pull/2000

🥺 , I wanted to validate if things were working fine! 🙏